scripts/data/ace-event/parse_ace_event.py

```diff
@@ -269,7 +268,7 @@ def get_token_of(doc, char):
 
 class Document:
     def __init__(self, annotation_path, text_path, doc_key, fold, heads_only=True,
-                 real_entities_only=True, include_pronouns=False, include_entity_coreference=False, include_event_coreference=False):
+                 real_entities_only=True, include_pronouns=False, include_entity_coreference=False, include_event_coreference=False, multitoken_triggers=False):
```

Comment:
I make a note about this in `document.py`: It looks like you're setting `multitoken_trigger` to `False` by default, which means this code (run in default mode) won't be compatible with the updated modeling code. Is that correct? I'm fine with just setting `multitoken_trigger` to `True` by default.

Reply:
Haha OK no worries. If getting your code merged in is gonna be too much of a pain, a reasonable alternative would just be for me to create a permanent `multitokentrigger` branch of `dygie` that people can use if they want to, and I can mention it in the docs. Let me know what you want to do.